src/components/Person/Person.jsx

```diff
@@ -1 +1,31 @@
-// export const Person = ({ person }) => ();
+const SEX_MALE = 'm';
+
+export const Person = ({ person }) => {
+  const { name, age, partnerName, sex, isMarried } = person;
+
+  const getPartnerInfo = () => {
+    if (isMarried) {
+      return `${partnerName} is my ${sex === SEX_MALE ? 'wife' : 'husband'}`;
+    }
+
+    return 'I am not married';
+  };
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {typeof age === 'number' && (
```

Comment:
Will be better to use `Boolean()` function instead of checking the type of the argument
We'll check the variable type further using Typescript
Let's check the value here

Also for now if age will be equal to 0, it will show the result on the page